builder/src/state/FileUpload.ts

```diff
@@ -49,6 +53,23 @@ export class FileUpload {
         }
     }
 
+    @action
+    private logErrors(...errors: string[]) {
+        this.uploadErrors.push(...errors);
+        console.error(errors);
+    }
+
+    @action
+    private logApiError(error: Error | ThunderstoreApiError) {
+        if (error instanceof ThunderstoreApiError) {
+            const flattened = stringifyError(error.errorObject);
+            this.logErrors(...flattened);
+        } else {
+            this.logErrors(error.message);
+            console.error(error, error.stack);
```

Comment:
Just checking if this logging is intentional since `logErrors` above already does some logging.

Reply:
Yeah since I'm not sure if `error.message` is actually super informative in the console. The previous code used `console.error(error, error.stack);` but that might be a bit too explicit to show on the UI itself, so I figured it'd make sense to keep logging it to the console.